bpmsuite/bpmsuite70-businesscentral-ephemeral.json

```diff
@@ -0,0 +1,349 @@
+{
+    "kind": "Template",
+    "apiVersion": "v1",
+    "metadata": {
+        "annotations": {
+            "description": "Application template for Red Hat JBoss BPM Suite 7.0 which includes only a Business Central.",
+            "iconClass": "icon-jboss",
+            "tags": "bpmsuite,jboss,xpaas",
+            "version": "1.0.0",
+            "openshift.io/display-name": "Red Hat JBoss BPM Suite 7.0 Business Central",
+            "openshift.io/image.insecureRepository": "true"
+        },
+        "name": "bpmsuite70-businesscentral-ephemeral"
+    },
+    "labels": {
+        "template": "bpmsuite70-businesscentral-ephemeral",
+        "xpaas": "1.4.0"
```

Comment:
I'm not sure what this version number should be, so I copied it from the previous templates